Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -449,18 +543,18 @@ bool TryInitialize() {
             });
 
             // If the singleton session and configuraiton have not yet been created, create them here now.
```

Comment:
Is this comment out of date now? I assume the "If" is referring to the condition you removed?

Reply:
Yes, will fix.